cli/printer/text.go

```diff
@@ -237,6 +238,28 @@ func (t *TextPrinter) PrintLoadBalancerHealthCheck(loadBalancer *models.LoadBala
 	return nil
 }
 
+func (t *TextPrinter) PrintLoadBalancerIdleTimeout(loadBalancer *models.LoadBalancer) error {
```

Comment:
ditto - i don't see this being used anywhere.

Reply:
See above comment.